### PR TITLE
Switch RPC to use String types where necessary

### PR DESCRIPTION
**Pull request by waynenilsen**

### CODE REVIEW

File: integration/util/scenario/validator.js [hunk 1]
```diff
@@ -166,9 +188,23 @@ class Validator {
     });
 
     const wsProvider = new WsProvider(`ws://localhost:${this.wsPort}`);
+    const types = await loadTypes(this.ctx);
     const api = await ApiPromise.create({
       provider: wsProvider,
-      types: await loadTypes(this.ctx)
+      types: {
+        ...types,
+        ApiAssetData: {
```
Comment (waynenilsen) [inline integration/util/scenario/validator.js]: dev note - not sure about this one .. just matching castle here

**Review by torreyatcitty — APPROVED** [review]
lgtm

File: integration/util/scenario/validator.js (hunk 1 above)
Comment (mykelp) [reply integration/util/scenario/validator.js]: removed.

**Review by jflatow — APPROVED** [review]
💪

**Pull request merged**